src/CryptoFraudDetection/scraper/twitter.py

```diff
@@ -7,3 +7,304 @@
+
+
+    def load_cookies(self, driver: webdriver.Firefox) -> None:
+        """
+        Loads cookies from a file and refreshes the page. If the file is not found or an error occurs, 
+        cookies are loaded from an environment variable.
+
+        Args:
+            driver (webdriver.Firefox): Selenium WebDriver instance.
+        """
+        driver.get("https://www.x.com")
+        
+        cookie_file_path = '../data/cookies_x_1_0.json'
+        
+        try:
+            with open(cookie_file_path, 'r') as file:
+                cookies = json.load(file)
+            print(f"Cookies successfully loaded from {cookie_file_path}")
+        except (FileNotFoundError, json.JSONDecodeError) as e:
+            # Fallback: Cookies aus der Umgebungsvariable laden, wenn das Cookiefile fehlt oder ungültig ist
+            print(f"Error loading cookies from file: {e}. Trying to load from environment variable.")
+            cookie_file_content = os.getenv('COOKIE_FILE_CONTENT_X')
+
+            if cookie_file_content:
+                try:
+                    cookies = json.loads(cookie_file_content)
+                    print("Cookies successfully loaded from environment variable.")
+                except json.JSONDecodeError as e:
+                    raise ValueError("Invalid JSON format in COOKIE_FILE_CONTENT_X environment variable.") from e
+            else:
+                raise FileNotFoundError(f"Neither cookie file '{cookie_file_path}' nor 'COOKIE_FILE_CONTENT_X' environment variable could provide cookies.")
+
+        
+        for cookie in cookies:
+            driver.add_cookie(cookie)
+
+        
+        driver.refresh()
+        time.sleep(10)
+
+
+    def navigate_to_explore(self, driver: webdriver.Firefox) -> None:
+        """
+        Navigates to the Twitter explore page and handles any popups.
+
+        Args:
+            driver (webdriver.Firefox): Selenium WebDriver instance.
+        """
+        driver.get("https://www.x.com/explore")
+        time.sleep(10)
+        print("Page title after loading cookies and navigating to Explore:", driver.title)
+
+        try:
+            close_button_wait = WebDriverWait(driver, 5)
+            close_button = close_button_wait.until(
+                EC.element_to_be_clickable((By.XPATH, "//button[contains(@aria-label, 'Close')]"))
+            )
+            driver.execute_script("arguments[0].click();", close_button)
+            print("Close button clicked successfully.")
+        except Exception as e:
+            self.logger.warning("Could not find or click the close button:", e)
+
+
+    def perform_search(self, driver: webdriver.Firefox, search_query: str) -> None:
+        """
+        Performs a search for the given query.
+
+        Args:
+            driver (webdriver.Firefox): Selenium WebDriver instance.
+            search_query (str): The search query to look for.
+        """
+        try:
+            search_bar = WebDriverWait(driver, 10).until(
```

Comment:
here you use a timeout of 10s and in the other options 5s, maybe create a variable or set it to 10 everywhere